opencl/CMakeLists.txt

```diff
@@ -44,7 +44,7 @@ FetchContent_GetProperties(ocl-headers)
 set(OpenCL_INCLUDE_DIR
   ${ocl-headers_SOURCE_DIR} CACHE PATH "Path to OpenCL Headers")
 
-target_compile_definitions(Headers INTERFACE -DCL_TARGET_OPENCL_VERSION=220)
+target_compile_definitions(Headers INTERFACE -DCL_TARGET_OPENCL_VERSION=300)
```

Comment:
I don't think that this change is necessary. We can still built `opencl-aot` using 2.2 version of headers. Technically, the tool is only likely using 1.2 API

Reply:
We discussed this offline: we need to keep the change so that runtime itself is able to see definitions for OCL 3.0.